skill-package/interactionModels/custom/it-IT.json

```diff
@@ -0,0 +1,143 @@
+{
```

Comment:
In order to have all Italian speech texts a file inside lambda/skill/i18n is needed as well. That would be amazing if you could add this as well. You can create a file similar to those that are already in the folder and add the Italian text.

Reply:
Solved by [3fdb2c4](https://github.com/LorenzHW/Telegram-Connect/pull/5/commits/3fdb2c464e3736c612162e9bc3ca097bf50b77fb)